docs/specs/tripgo.swagger.yaml

```diff
@@ -896,7 +896,91 @@ paths:
         required: true
         description: Identifier of trip as returned by previous server call.
 
-  /trip/{id}/planned:
+  /trip/hook/{id}:
+    post:
+      tags:
+      - Trips
+      summary: Hooks a trip to real-time updates
+      description:
+        Hooks a previously computed trip to get updates whenever is updated due to real-time data sources.
+
+        Rather than constructing this URL directly, you should get it from the `hookURL` property from a computed trip.
+      parameters:
+      - name: input
+        in: body
+        schema:
+          type: object
+          properties:
+            url:
+              type: string
+              description: The webhook url to be notified when the trips is changed due to real-time updates. When that happens, our servers will POST to the registered URL a JSON object with the `tripID` and the `tripURL` to be retrieved.
```

Comment:
Would be nice if it also posted some kind of enum indicating the change, e.g., "TRIP_UPDATED" or "TRIP_CANCELED"

Reply:
not sure what you mean by trip_canceled, something like Uber driving cancelling the ride?

Reply:
Yes, or you planning a train trip and then your train getting canceled.